thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -167,18 +148,14 @@ export function toBaselineUrn(urn) {
   return metricUrnHelper('frontend:metric:baseline:', urn);
 }
 
-export function toOffsetUrn(urn, compareMode) {
-  return metricUrnHelper(`frontend:metric:${compareMode}:`, urn);
-}
-
 export function toMetricUrn(urn) {
   return metricUrnHelper('thirdeye:metric:', urn);
 }
 
 export function toMetricLabel(urn, entities) {
-  let metricName = urn;
-  if (entities && entities[urn]) {
-    metricName = entities[urn].label.split("::")[1].split("_").join(' ');
+  let metricName = stripTail(urn);
+  if (entities && entities[stripTail(urn)]) {
```

Comment:
can't we just reuse `metricName` here? `entities[metricName]` and `metricName = entities[metricName].label...`

Reply:
For all comments, please also see the new version of this at https://github.com/linkedin/pinot/pull/2351